byron/ledger/executable-spec/src/Ledger/Delegation.hs

```diff
@@ -185,11 +195,15 @@ data DSEnv = DSEnv
   { _dSEnvAllowedDelegators :: Set VKeyGenesis
   , _dSEnvEpoch :: Epoch
   , _dSEnvSlot :: Slot
-  , _dSEnvStableAfter :: BlockCount
   } deriving (Show, Eq)
 
 makeFields ''DSEnv
 
+-- TODO(md) 2019-04-30: confirm this Byron-era value with the team
+-- | Chain stability parameter
+k :: BlockCount
+k = BlockCount 10
```

Comment:
This is `2160` in Byron and multiplied by `10` to get the `epochSlots = 21600`

Reply:
Done!